core/bones/recordBone.py

```diff
@@ -56,6 +56,10 @@ def parseSubfieldsFromClient(self) -> bool:
 	def singleValueFromClient(self, value, skel, name, origData):
 		usingSkel = self.using()
 		usingSkel.fromClient(value)
+
+		# Discard any errors from non-required bones
```

Comment:
When this is not done, ViUR does not accept any edit/add action containing a recordBone that contain an empty, non-required bone.

Reply:
I was able to reproduce this. Suppressing the errors in this way would f.e allow you to store any value in an email bone as long it's not required. We'll have to fix this in a different way (probably making bone.fromClient the errors as well as a overall success/failure marker)

Reply:
Nope, because the emailBone doesn't take the value provided. It is thrown away in this case. I'm a little confused why there is still an error message for bones which are allowed to be empty. IMHO this is the confusing part, and if this won't be available, I can remove this line and take all real errors which are errors as they are.

Reply:
Yes, pls remove these lines; I'll address the issue with empty non-required fields in a required container next week in a separate pr. The rest of this PR should be resolved.

Reply:
OK I removed the discarding from the PR.